src/components/ContactInfo/ContactInfo.vue

```diff
@@ -103,21 +105,17 @@
 
     <!-- Edit Contact Info -->
     <template v-else id="edit-contact-form">
-      <v-row class="mx-0">
-        <v-col cols="3" class="pl-0 py-0">
+      <v-row class="mx-0 my-1" no-gutters>
+        <v-col cols="3">
           <label :class="{'error-text': invalidSection}">
             {{ contactLabel }} Contact Information
           </label>
         </v-col>
-      </v-row>
-
-      <v-row class="mx-0">
-        <v-col cols="3"></v-col>
-        <v-col cols="9" v-if="customMsg != null" class="my-4 info-text pl-0 py-0">
+        <v-col cols="9" v-if="customMsg != null" class="info-text pl-0 py-0">
```

Comment:
So the custom message is now on the same row? Can you post before and after screenshots? Since this is a shared component, I'm wondering if this will be OK in all places it's used.

Reply:
Definitely, 1s!

Reply:
![Screen Shot 2022-03-02 at 12 22 53 PM](https://user-images.githubusercontent.com/53542131/156443561-871dce17-9a53-4f42-b971-d43da0bb8f92.png)
![Screen Shot 2022-03-02 at 12 22 02 PM](https://user-images.githubusercontent.com/53542131/156443564-70f77199-061b-49e3-8329-036fe7d28146.png)

Reply:
Pretty straight forward! I believe this should be pretty low impact :)

Reply:
I can't believe we had it the other way. Looks great now.

Reply:
hahaha 😆 
Agreed!